.github/workflows/release.yml

```diff
@@ -17,6 +17,9 @@ on:
         description: "Release version (v#.#.#[-rc#])"
         required: true
 
+permissions:
+  contents: write
```

Comment:
This allows us to get rid of the special `GHA_GIT_COMMIT` token

Reply:
This is correct. There is an option to control it such that the permission is only on specific steps instead the entire workflow, but I figured this is the best 1:1 swap for now.